### ISSUE TITLE
Feature Request: Support for Multi-channel Routing

### ISSUE THREAD

**Issue by carltesta** [essej/sonobus#15]
I'm currently using Sonobus on Linux with SuperCollider. I'd like to send SonoBus participants' audio through JACK into SuperCollider for processing and then send the audio back into SonoBus so that they can hear the processed audio. I can do this with JackTrip and the JackRouter but I'd like to try this with SonoBus as it can be easier for some folks to use. Is it possible to support a channel assignment for each participant in the SonoBus group? If each participant showed up as a JACK input/output then it would be easy to route audio where it needs to go. Otherwise some way to activate more than two audio outputs at a time in SonoBus would be helpful. Thanks!

**Comment by essej** [comment]
Yes, I would like to have this ability too, but I will need to hack at JUCE's Jack support, which is not well thought out for how JACK actually works. In the meantime, the VST3 plugin (which you can also build in Linux with 'make VST3') has full multi-bus output support right now... so if you can a load native VST3 in any host you have, you can get it working now. I have "fix JACK support" on my TODO list...

**Comment by Spacechild1** [comment]
> I'd like to send SonoBus participants' audio through JACK into SuperCollider for processing and then send the audio back into SonoBus so that they can hear the processed audio.

Why don't you use the Sonobus VST3 plugin directly in Supercollider with the VSTPlugin extension? ;-)

**Comment by carltesta** [comment]
> Why don't you use the Sonobus VST3 plugin directly in Supercollider with the VSTPlugin extension? ;-)

For the particular use case I'm thinking of I'm running SuperCollider running on Linux. To my understanding the VSTPlugin extension wouldn't be able to run the native SonoBus GUI on Linux, is that right?

**Comment by Spacechild1** [comment]
> To my understanding the VSTPlugin extension wouldn't be able to run the native SonoBus GUI on Linux, is that right?

VSTPlugin has always supported the plugin GUI on Linux and Windows. The problem was with macOS, but since SC 3.11 it also works there.

**Comment by carltesta** [comment]
Now the question becomes: is there a build of the VST plugin on Linux? The SonoBus description only lists VSTs for Mac and Windows. I think I could build it if I know the right flags to use when building.
